iothub/device/src/Transport/AmqpIot/AmqpIotTransportLog.cs

```diff
@@ -12,259 +12,207 @@ internal class AmqpIotTransportLog : AmqpTrace
         public override void AmqpAbortThrowingException(string exception)
         {
             if (Logging.IsEnabled)
-            {
                 Logging.Info(exception, "AmqpAbortThrowingException:", "AmqpTransportLog");
-            }
 
             base.AmqpAbortThrowingException(exception);
         }
 
         public override void AmqpAddSession(object source, object session, ushort localChannel, ushort remoteChannel)
         {
             if (Logging.IsEnabled)
-            {
                 Logging.Associate(source, session, "AmqpAddSession:", $"source:{source}.session:{session}.localChannel:{localChannel}.remoteChannel:{remoteChannel}");
```

Comment:
It appears that we have several long logging lines within this class. Can we separate into multiple lines for each of them?